src/globalFilters/GlobalFilterSelector.vue

```diff
@@ -94,7 +96,7 @@ export default {
     openOverlay() {
       this.overlay = this.openmct.overlays.overlay({
         element: this.$el,
-        size: 'small',
```

Comment:
yeah I noticed small seems to have some styling issues. Good change.

Reply:
Yeah 'small' was missing the border padding but fit seems better for this in general since it'll scale with the number of filters you use (which I expect to always be small - less than 3 per project most likely)